microsoft-edge/webview2/get-started/win32.md

```diff
@@ -55,6 +55,7 @@ Add the WebView2 SDK into the project.  Use NuGet to install the Win32 SDK.
     :::image-end:::
 
 1.  Install the Windows Implementation Library.
+    1.  In the **Nuget** window click the **Browse** tab.
```

Comment:
Global: change to NuGet cap'zn per https://www.nuget.org/

Reply:
fixing